backend/main_test.go

```diff
@@ -27,6 +31,12 @@ import (
 	"testing"
 )
 
+func makeHandler() http.Handler {
+	server := Server{store: in_memory_storage.MakeInMemStore()}
+	server.auth = noAuth{}
```

Comment:
Again, a nit: can you just pass this directly in when you instantiate the `Server` struct? I.e.

`server := Server{store: ..., auth: ...}`

Reply:
Yep, makes perfect sense, don't know why that didn't cross my mind 😅